Workbooks/WorkbooksMetadata.json

```diff
@@ -701,7 +740,47 @@
     "subtitle": "",
     "provider": "Edoardo Gerosa"
   },
-  {    "workbookKey": "GitHubSecurityWorkbook",
+  {
+    "workbookKey": "VMwareCarbonBlackWorkbook",
+    "logoFileName": "vmwarecarbonblack_logo.svg",
+    "description": "Gain extensive insight into VMware Carbon Black Cloud - Endpoint Standard by analyzing, collecting and correlating Event logs.\nThis workbook provides visibility into Carbon Black managed endpoints and identified threat event",
+    "dataTypesDependencies": [ "CarbonBlackEvents_CL","CarbonBlackNotifications_CL","CarbonBlackAuditLogs_CL" ],
+    "dataConnectorsDependencies": [ "VMwareCarbonBlack" ],
+    "previewImagesFileNames": [ "VMwareCarbonBlackWhite.png", "VMwareCarbonBlackBlack.png" ],
```

Comment:
I suspect the term CarbonBlack would conflict with the Black and White png names. Please rename the png files for just this one to VMWareCarbonBlack.png and VMWareCarbonWhite.png and change in metadata file as well. That way there's less logic confusion.

Reply:
corrections have been made and changes pushed.